### DISCUSSION TITLE
Shlink client for Android

### DISCUSSION THREAD

**Discussion by cyb3rko** [shlinkio/discussions#46] [General]
Hey,
I'm starting to work on a native Android client for Shlink.

1. Am I allowed to call it just "Shlink" or something like "Shlink Client"?
2. Am I allowed to use the Shlink logo or do I need to use any other logo?

Thanks!

**Comment by acelaya** [comment]
Hey!

Yeah, feel free to use the logo and name. Just make sure to mention it's an unofficial app.

Thanks!

**Reply by cyb3rko** [reply]
Perfect, thanks a lot.

Yeah, of course.
